src/components/backend-ai-resource-preset-list.ts

```diff
@@ -331,10 +345,10 @@ class BackendAiResourcePresetList extends BackendAIPage {
 
   firstUpdated() {
     this.notification = globalThis.lablupNotification;
-    const textfields = this.shadowRoot.querySelectorAll('mwc-textfield');
-    for (const textfield of textfields) {
+    const textfields = this.shadowRoot?.querySelectorAll('mwc-textfield');
+    textfields?.forEach((textfield) => {
       this._addInputValidator(textfield);
-    }
+    });
   }
 
```

Comment:
I'm not sure whether this change affected textfield vanishing in creating/modifying resource preset.
I already confirmed it's happening in Chrome (version 103.0.5060.114), and Safari (version 15.5) browser.

## Screenshots
![Screen Shot 2022-07-20 at 2 27 22 AM](https://user-images.githubusercontent.com/46954439/179812995-b7e9ddfd-42f5-486d-bcad-3bf739322748.png)
![Screen Shot 2022-07-20 at 2 27 16 AM](https://user-images.githubusercontent.com/46954439/179812982-3ebdee17-9043-4675-909a-5abd04a2ec54.png)